backend/src/main/resources/db/migration/internal/V0.116__update_infraction_vessel_size_type.sql

```diff
@@ -0,0 +1,26 @@
+
+UPDATE env_actions t
+SET value = 
+jsonb_set(value, '{infractions}', 
+		  (SELECT jsonb_agg(
+		CASE
+			WHEN j1->'vesselSize' = '"LESS_THAN_12m"'
+			THEN jsonb_set(j1, '{vesselSize}','"11"')
```

Comment:
`11`plutot que `"11"`pour un nombre, non ? si oui, idem pour les autres plus bas.
```suggestion
			THEN jsonb_set(j1, '{vesselSize}','11')
```